packages/to-json-schema/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@valibot/to-json-schema",
+  "description": "Valibot to JSON schema converter",
+  "version": "0.1.0",
+  "license": "MIT",
+  "author": "Fabian Hiller",
+  "homepage": "https://valibot.dev",
+  "contributors": [
+    {
+      "name": "Guillaume Cornut"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fabian-hiller/valibot"
+  },
+  "keywords": [
+    "valibot",
+    "schema",
+    "converter",
+    "JSON schema"
+  ],
+  "type": "module",
+  "main": "./index.js",
+  "types": "./index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./index.d.ts",
+        "default": "./index.js"
+      },
+      "require": {
+        "types": "./index.d.cts",
+        "default": "./index.cjs"
+      }
+    }
+  },
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "vitest --typecheck",
+    "coverage": "vitest run --coverage --isolate",
+    "lint": "eslint \"src/**/*.ts*\" && tsc --noEmit",
+    "eslint.fix": "eslint \"src/**/*.ts*\" --fix",
+    "format": "prettier --write ./src",
+    "format.check": "prettier --check ./src",
+    "build": "tsup"
+  },
+  "devDependencies": {
+    "@types/eslint": "^8.56.10",
+    "@types/json-schema": "^7.0.15",
+    "@types/node": "^20.12.2",
+    "@typescript-eslint/eslint-plugin": "^7.15.0",
+    "@typescript-eslint/parser": "^7.15.0",
+    "@vitest/coverage-v8": "^1.6.0",
+    "eslint": "^8.57.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsdoc": "^48.5.2",
+    "eslint-plugin-security": "^2.1.1",
+    "tsup": "^8.1.0",
+    "typescript": "^5.5.3",
+    "valibot": "workspace:*",
```

Comment:
Depending on the workspace version of valibot here is convenient but it actually produces errors (in IDE, during type check, test, build) when the valibot does not have a built `dist` folder (the default state when cloning the repo)

I initially added `paths` in the tsconfig to tell TS & vite to use the raw source files instead.

One other solution could be to always build valibot before running anything in this package. 

Or we could just use valibot from NPM instead of the workspace but it would need to be updated at each new valibot release.

What do you think would be the best solution here ? 🤔

Reply:
Sorry, that was my fault. Please undo my changes. I will ask next time.